beacon-chain/db/restore_test.go

```diff
@@ -65,7 +66,7 @@ func TestRestore(t *testing.T) {
 	require.NoError(t, err)
 	headBlock, err := restoredDb.HeadBlock(ctx)
 	require.NoError(t, err)
-	assert.Equal(t, uint64(5000), headBlock.Block.Slot, "Restored database has incorrect data")
+	assert.Equal(t, types.Slot(5000), headBlock.Block.Slot, "Restored database has incorrect data")
```

Comment:
@farazdagi not sure how this was passing for bazel

Reply:
That's really strange!